examples/defi/auto-lend/tests/lib.rs

```diff
@@ -6,28 +6,30 @@ use auto_lend::User;
 
 struct TestEnv<'a, L: SubstateStore> {
     executor: TransactionExecutor<'a, L>,
-    key: EcdsaPublicKey,
-    account: Address,
-    usd: Address,
-    lending_pool: Address,
+    public_key: EcdsaPublicKey,
+    account: ComponentId,
+    usd: ResourceDefId,
```

Comment:
`ResourceId` feels much better

Reply:
Haha, I do like `ResourceId`. Let's revisit the terminologies in LA; we have a dedicated session for that :)